clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/LeaderFollowerStoreIngestionTask.java

```diff
@@ -3600,28 +2911,15 @@ protected void processVersionSwapMessage(
     }
   }
 
-  protected LeaderProducerCallback createProducerCallback(
-      PubSubMessage<KafkaKey, KafkaMessageEnvelope, Long> consumerRecord,
-      PartitionConsumptionState partitionConsumptionState,
-      LeaderProducedRecordContext leaderProducedRecordContext,
-      int partition,
-      String kafkaUrl,
-      long beforeProcessingRecordTimestampNs) {
-    return new LeaderProducerCallback(
-        this,
-        consumerRecord,
-        partitionConsumptionState,
-        leaderProducedRecordContext,
-        partition,
-        kafkaUrl,
-        beforeProcessingRecordTimestampNs);
-  }
-
   protected Lazy<VeniceWriter<byte[], byte[], byte[]>> getVeniceWriter(
       PartitionConsumptionState partitionConsumptionState) {
     return partitionConsumptionState.getVeniceWriterLazyRef();
   }
 
+  protected void setRealTimeVeniceWriterRef(PartitionConsumptionState partitionConsumptionState) {
+    partitionConsumptionState.setVeniceWriterLazyRef(veniceWriterForRealTime);
+  }
```

Comment:
Do we need functions for this? Why not just call `partitionConsumptionState.getVeniceWriterLazyRef()` directly wherever we need to get the VW? And same for the setter?